modules/forum/src/main/model.scala

```diff
@@ -56,9 +56,18 @@ case class MiniForumPost(
     topicName: String,
     userId: Option[UserId],
     text: String,
-    createdAt: Instant
+    createdAt: Instant,
+    categId: ForumCategId
 )
 
+class RecentForumTopic(minis: List[MiniForumPost]):
+  val posts     = minis.sortBy(_.createdAt)(Ordering[Instant])
+  val name      = posts.head.topicName
+  val categId   = posts.head.categId
```

Comment:
please don't use `.head`, it's unsafe and will break the entire page if/when `posts` is empty.

If it can't happen, then prove it with a type: `NonEmptyList[MiniForumPost]`. Then using `.head` is safe and sound.